scipy/special/cephes/mconf.h

```diff
@@ -100,7 +99,12 @@
 #ifndef cephes_isnan
 #define cephes_isnan(x) isnan(x)
 #define cephes_isinf(x) isinf(x)
-#define cephes_isfinite(x) npy_isfinite(x)
+#define cephes_isfinite(x) isfinite(x)
 #endif
 
+/* Constants needed that are not available in the C standard library */
+#define SCIPY_EULER     0.577215664901532860606512090082402431   /* Euler constant */
+#define SCIPY_El        2.718281828459045235360287471352662498L  /* e as long double */
+#define SCIPY_LOGE2     0.693147180559945309417232121458176568   /* log_e 2 */
```

Comment:
With [Microsoft compilers](https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170) and [GNU compilers](https://www.gnu.org/software/libc/manual/html_node/Mathematical-Constants.html), this is `M_LN2`.  Can we use that instead of defining `SCIPY_LOGE2`?

Reply:
Good catch. Yes, I think so - will update.